types/index.d.ts

```diff
@@ -0,0 +1 @@
+// Minimum TypeScript Version: 5.0
```

Comment:
@MysteryBlokHed This fixes dtslint, which I think is now returning only real errors.